salt_stack/salt/lambda-dev/files/makedomainenv

```diff
@@ -1,125 +1,86 @@
    export ATLAS=./usr/lib64/atlas/libatlas.so
    export BLAS=./usr/lib64/atlas/libptf77blas.so
    export LAPACK=./usr/lib64/atlas/liblapack.so
 
-   # Creating env and usr files to help debug when virtualenv fails to install correctly.
-   find /usr -print > ~/debug/usr-before-${STARTTIME}.txt
-
    if [ -d ~/virtualenvs/$1 ]; then
       echo delete previous virtualenv for $1
       rm -rf ~/virtualenvs/$1
    fi
-   python3 -m virtualenv ~/virtualenvs/$1 
+
+   python3 -m virtualenv ~/virtualenvs/$1
    source ~/virtualenvs/$1/bin/activate
 
    # copy the BLAS libraries into relative directories
    mkdir ~/virtualenvs/$1/lib64/atlas
-   cp /usr/lib64/atlas/libatlas.so ~/virtualenvs/$1/lib64/atlas
-   cp /usr/lib64/atlas/libptf77blas.so ~/virtualenvs/$1/lib64/atlas
-   cp /usr/lib64/atlas/liblapack.so ~/virtualenvs/$1/lib64/atlas
+   cp /usr/lib64/atlas/libatlas.so.3 ~/virtualenvs/$1/lib64/atlas
+   cp /usr/lib64/atlas/libptf77blas.so.3 ~/virtualenvs/$1/lib64/atlas
+   cp /usr/lib64/atlas/libf77blas.so.3 ~/virtualenvs/$1/lib64/atlas
+   cp /usr/lib64/atlas/libptcblas.so.3 ~/virtualenvs/$1/lib64/atlas
+   cp /usr/lib64/atlas/libcblas.so.3 ~/virtualenvs/$1/lib64/atlas
+   cp /usr/lib64/atlas/liblapack.so.3 ~/virtualenvs/$1/lib64/atlas
 
-   #workon $1
-   env > ~/debug/env-before-${STARTTIME}.txt
-   
-   #pip install --upgrade pip
+   cp /usr/lib64/libgfortran.so.3 ~/virtualenvs/$1/lib64/atlas
+   cp /usr/lib64/libquadmath.so.0 ~/virtualenvs/$1/lib64/atlas
 
+   cd ~/virtualenvs/$1/lib/python3.6/site-packages
 
-   cd ~/virtualenvs/$1/lib/python3.4/site-packages
    # cloud formation config should have already zipped spdb, lambda, lambda-utils and bossutils from local machine.
-   unzip /home/ec2-user/sitezips/${1}.zip -d /home/ec2-user/virtualenvs/$1/lib/python3.4/site-packages
-   #unzip /home/ec2-user/sitezips/${1}.zip -d /home/ec2-user/virtualenvs/$1/local/lib/python3.4/site-packages
-
-   #cp -r ~/spdb $WORKON_HOME/$1/local/lib/python3.4/site-packages/spdb.git
-
-   #cd ~/virtualenvs/$1/local/lib/python3.4/site-packages
+   unzip /home/ec2-user/sitezips/${1}.zip -d /home/ec2-user/virtualenvs/$1/lib/python3.6/site-packages
 
    # set the correct log config
    echo configuring logging
-   cd ~/virtualenvs/$1/local/lib/python3.4/site-packages/bossutils/
+   cd ~/virtualenvs/$1/local/lib/python3.6/site-packages/bossutils/
    mv ./lambda_logger_conf.json ./logger_conf.json
 
-   cd ~/virtualenvs/$1/lib/python3.4/site-packages
+   cd ~/virtualenvs/$1/lib/python3.6/site-packages
 
    # Fix ndingest folder name.
    mv ndingest.git ndingest
 
    # Fix spdb folder name.
    mv spdb.git spdb
    cd spdb
-   pip install -r requirements.txt
+   pip install -t ~/virtualenvs/$1/local/lib/python3.6/dist-packages -r requirements.txt
 
    cd c_lib/c_version
    cp makefile_LINUX makefile
    make
 
-   # (SH) When virtualenv fails, numpy is not listed in freeze.  This check will help debug the problem when it occurs again.
```

Comment:
So this is no longer a concern?

Reply:
Yep. Using `-t` when doing `pip install` forces numpy into the "proper" location.

Reply:
Sweet